seirmo/_stoch_model.py

```diff
@@ -0,0 +1,59 @@
+#
+# This file is part of seirmo (https://github.com/SABS-R3-Epidemiology/seirmo/)
+# which is released under the BSD 3-clause license. See accompanying LICENSE.md
+# for copyright notice and full license details.
+#
+
+import numpy as np
+import seirmo as se
+from ._gillespie import solve_gillespie
+
+
+class StochasticSEIRModel(se.SEIRForwardModel):
+    def __init__(self, params_names: list):
+        super(StochasticSEIRModel, self).__init__()
+        self._parameters = se.SEIRParameters(params_names)
+        # sets up n compart and param names output are the variables
+        # we want to look at like S, E etc, compartment pops
```

Comment:
If I'm honest, this is because we found this code confusing to implement. The deterministic class has a method to input the parameter names you want to output (with the default set to all), and so we wanted to replicate this functionality in our class, however found it confusing that this default is (in effect) 'hidden'. 
I've clarified the comments, but do say if you'd rather the functionality here is changed?

Reply:
Brilliant thanks!